src/data/mockData.js

```diff
@@ -1,5 +1,5 @@
 export const mockEvents = {
-  "data": [
+  data: [
     {
       id: "1",
       type: "event",
```

Comment:
Good catch here! I didn't even realize I added those quotes. 😄 Great work having such an eye for detail.